Dockerfile

```diff
@@ -0,0 +1,10 @@
+# From Agora Runner
+FROM bpfk/agora:latest
+RUN apk add --no-cache curl git python3 py-pip alpine-sdk \
+    bash autoconf libtool automake nodejs npm supervisor
+ADD . /root/stoa/
+WORKDIR /root/stoa/
+RUN npm ci
+EXPOSE 3836
+# Use the supervisord to run agora and stoa multiprocessor.
+ENTRYPOINT ["supervisord","-c","/root/stoa/config/service_script.conf"]
```

Comment:
It's better not to install it in the root directory. Use `/stoa/` for the moment (like we use `/agora/`)

Reply:
Yes, I fixed it now.